src/main/java/com/jagrosh/jmusicbot/commands/dj/PlaynextCmd.java

```diff
@@ -18,111 +18,48 @@
 import com.jagrosh.jdautilities.command.CommandEvent;
 import com.jagrosh.jmusicbot.Bot;
 import com.jagrosh.jmusicbot.audio.AudioHandler;
-import com.jagrosh.jmusicbot.audio.QueuedTrack;
-import com.jagrosh.jmusicbot.commands.DJCommand;
+import com.jagrosh.jmusicbot.commands.PlayCommand;
 import com.jagrosh.jmusicbot.utils.FormatUtil;
-import com.sedmelluq.discord.lavaplayer.player.AudioLoadResultHandler;
-import com.sedmelluq.discord.lavaplayer.tools.FriendlyException;
-import com.sedmelluq.discord.lavaplayer.track.AudioPlaylist;
 import com.sedmelluq.discord.lavaplayer.track.AudioTrack;
+import com.jagrosh.jmusicbot.audio.QueuedTrack;
+
 import net.dv8tion.jda.core.entities.Message;
 
 /**
  *
  * @author John Grosh (john.a.grosh@gmail.com)
  */
-public class PlaynextCmd extends DJCommand
-{
-    private final String loadingEmoji;
-    
-    public PlaynextCmd(Bot bot, String loadingEmoji)
-    {
-        super(bot);
-        this.loadingEmoji = loadingEmoji;
+public class PlaynextCmd extends PlayCommand {
+    public PlaynextCmd(Bot bot, String loadingEmoji) {
+        super(bot, loadingEmoji);
         this.name = "playnext";
-        this.arguments = "<title|URL>";
         this.help = "plays a single song next";
-        this.beListening = true;
-        this.bePlaying = false;
     }
-    
+
     @Override
-    public void doCommand(CommandEvent event)
-    {
-        if(event.getArgs().isEmpty() && event.getMessage().getAttachments().isEmpty())
-        {
+    public void doCommand(CommandEvent event) {
+        if (event.getArgs().isEmpty() && event.getMessage().getAttachments().isEmpty()) {
             event.replyWarning("Please include a song title or URL!");
             return;
         }
-        String args = event.getArgs().startsWith("<") && event.getArgs().endsWith(">") 
-                ? event.getArgs().substring(1,event.getArgs().length()-1) 
-                : event.getArgs().isEmpty() ? event.getMessage().getAttachments().get(0).getUrl() : event.getArgs();
-        event.reply(loadingEmoji+" Loading... `["+args+"]`", m -> bot.getPlayerManager().loadItemOrdered(event.getGuild(), args, new ResultHandler(m,event,false)));
+        String args = getLoadingArgs(event);
+        event.reply(loadingEmoji + " Loading... `[" + args + "]`", m -> bot.getPlayerManager()
+                .loadItemOrdered(event.getGuild(), args, new ResultHandler(m, event, false)));
     }
-    
-    private class ResultHandler implements AudioLoadResultHandler
-    {
-        private final Message m;
-        private final CommandEvent event;
-        private final boolean ytsearch;
-        
-        private ResultHandler(Message m, CommandEvent event, boolean ytsearch)
-        {
-            this.m = m;
-            this.event = event;
-            this.ytsearch = ytsearch;
+
+    private class ResultHandler extends PlayResultHandler {
```

Comment:
I'm not really sure why you need this class; all it appears to do differently than the regular play result handler is not check for max duration